tests/spanner_dbapi/test_connect.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+"""connect() module function unit tests."""
+
+import mock
+import unittest
+
+
+def _make_credentials():
+    import google.auth.credentials
+
+    class _CredentialsWithScopes(
+        google.auth.credentials.Credentials, google.auth.credentials.Scoped
+    ):
+        pass
+
+    return mock.Mock(spec=_CredentialsWithScopes)
+
+
+class Testconnect(unittest.TestCase):
```

Comment:
Not `Test_connect`?

Reply:
Looks like `_` is used only because the function name itself starts with a `_` symbol: `_make_list_value_pbs`. Still, I don't think it's a strict naming rule, and I like `Test_connect` more. Let's rename